.environment/gitleaks/gitleaks-config.toml

```diff
@@ -0,0 +1,281 @@
+title = "PRIME ReportStream Gitleaks Configuration"
+
+# Global allowlist
+[allowlist]
+    description = "Allow-list for files and paths"
+    files = [
+        '(.*?)(bin|doc|gif|iml|jar|jp(e)?g|pdf|png|xlsx)$',
+        '^\.?gitleaks-config.toml$',
+        '^\.?gitleaks.report.json$',
+        '^package-lock\.json$',
+        'cleanslate.sh.log',
+        'yarn\.lock$',
+    ]
+    paths = [
+        '.idea/',
+        '.terraform/providers/',
+        'frontend-react/build/',
+        'frontend-react/node_modules/',
+        'prime-router/.gradle/',
+        'prime-router/build/',
+        'prime-router/docs/dependency-graph-full/dependency-graph-full.txt',
+        'prime-router/docs/schema_documentation/',
+        'prime-router/src/test/csv_test_files/input/',
+    ]
+
+[[rules]]
+    description = "Email"
+    regex = '[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}'
+    tags = [
+        "email"
+    ]
+    [rules.allowlist]
+        # TODO: Filter the real ones down
+        regexes = [
+            '(?i)@cdc.local',
+            '(?i)@email.com',
+            '(?i)@organization.tld',
+            '(?i)a@cdc.gov',
+            '(?i)adhelpdsk@cdc.gov',
+            '(?i)data@cdc.gov',
+            '(?i)e.ripley@weylandyutani.com',
+            '(?i)jbrush@avantecenters.com',
+            '(?i)jj@phd.gov',
+            '(?i)joe.jones@az.pima.gov',
+            '(?i)local@test.com',
+            '(?i)noreply@cdc.gov',
+            '(?i)prime@cdc.gov',
+            '(?i)qom6@cdc.gov',
+            '(?i)qtv1@cdc.gov',
+            '(?i)qva8@cdc.gov',
+            '(?i)reportstream@cdc.gov',
+            '(?i)support@simplereport.gov',
+            '(?i)usds@cdc.gov',
+            '(?i)usds@omb.eop.gov',
+        ]
```

Comment:
Looked up where these emails are. Interesting some are in docs. Not sure if we should have a client's email in a doc, but that's for a separate PR.